src/foam/u2/grid/GridCell.js

```diff
@@ -0,0 +1,349 @@
+foam.CLASS
+({
+    package: "foam.u2.grid",
```

Comment:
FOAM follows the Google coding standard of always using single quotes for strings, unless they actually contain a single quote character, so:

package: 'foam.u2.grid'

This is actually a useful convention because if I was searching for a 'foam.u2.grid', I would know that I needed to search for 'foam.u2.grid' and not "foam.u2.grid".